lib/supplejack/story.rb

```diff
@@ -209,18 +209,31 @@ def owned_by?(user)
       user.try(:api_key) == self.api_key
     end
 
-    def to_json
-      attributes.to_json
+    def as_json(include_contents: true)
```

Comment:
do you think we can replace this with:
`include_contents ? attributes : attribute.except(:contents)`